automatic/kubernetes-kompose/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,9 @@
+$ErrorActionPreference = 'Stop'
+
+$installLocation = "$(Get-ToolsLocation)\kubernetes-kompose"
+
+Write-Host "Removing Kompose from '$installLocation'"
+Remove-Item $installLocation -Recurse -Force -ea 0
+
+$newPath = $Env:Path.Replace(";$installLocation\bin", '')
```

Comment:
This will only change the path for the current session, when the user refreshes their session or creates a new session, path will still include the install location

Reply:
You're right, I will fix it; too bad there's no built-in convenience helper for this, the opposite to Install-ChocolateyPath.

Reply:
true, however, they are working on such an helper.
It just isn't ready just yet I believe:
https://github.com/chocolatey/choco/issues/310